src/BayesFilters/include/BayesFilters/WhiteNoiseAcceleration.h

```diff
@@ -34,7 +34,7 @@ class bfl::WhiteNoiseAcceleration : public StateModel
 
     void motion(const Eigen::Ref<const Eigen::MatrixXf>& cur_states, Eigen::Ref<Eigen::MatrixXf> prop_states) override;
 
-    Eigen::MatrixXf getNoiseSample(const int num) override;
+    Eigen::MatrixXf getNoiseSample(const int num);
```

Comment:
If it does not override, can we move it to `protected`?

Reply:
Of course!

Reply:
Great! 👍

Reply:
Now it is again with `override` since implements pure virtual method `AdditiveStateModel::getNoiseSample`.